src/Service.Host/client/src/components/Root.js

```diff
@@ -105,8 +105,8 @@ const Root = ({ store }) => (
                                 />
                                 <Route
                                     exact
-                                    path="/purchasing/reports/spend-by-supplier/report"
-                                    component={SpendBySupplier}
```

Comment:
How come this is getting overwritten?

Reply:
how did I manage that? Thanks. Good spot